telemetry/definitions/commonDefinitions.json

```diff
@@ -1,5 +1,15 @@
 {
     "types": [
+        {
+            "name": "acceptedFixType",
+            "type": "string",
+            "description": "Captures the type of fix that was accepted",
+            "allowedValues": [
+                "acceptDiff",
+                "insertAtCursor",
+                "copyDiff"
```

Comment:
Shouldn't this map to the buttons in the webview? If that's the case it should be `openDiff` instead of `acceptDiff`

Reply:
Corrected it and added quickFix as well to codeFixActions